### ISSUE TITLE
Determine fate of Login button for initial deployment

### ISSUE THREAD

**Issue by kerchner** [gwu-libraries/scholarspace#30]
Since regular users shouldn't be able to log in (yet).

**Comment by kilahimm** [comment]
If the following is added to the app/models/ability.rb file:

```
  def generic_file_abilities
    can :view_share_work, [GenericFile]
    can :create, [GenericFile] if admin_user?
  end
```

the create options are suppressed for all users except admin group members.

**Comment by kerchner** [comment]
As discussed today, we can distribute the login URL to administrators.

**Comment by kilahimm** [comment]
I'd recommend we also add that ability.rb file change i suggested above.  This way in case anyone finds there way /login they won't have any rights to do write/update changes to the system.

**Comment by kerchner** [comment]
Concur

**Comment by kilahimm** [comment]
Closing this since it was resolved by #50 for now.  Could be revisited per #67 in the future.
